api/db/changelog/db.changelog-167-add-access-module-tables.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-167-add-access-module-tables">
+    <createTable tableName="access_module">
+      <column name="name" type="ENUM('ERA_COMMONS', 'TWO_FACTOR_AUTH', 'RAS_LOGIN_GOV', 'RT_COMPLIANCE_TRAINING', 'DATA_USER_CODE_OF_CONDUCT', 'PROFILE_CONFIRMATION', 'PUBLICATION_CONFIRMATION')">
```

Comment:
Is there a way to define this enum first, so we don't have to repeat the list?

Reply:
did not find a good solution for that. Actually using enum as FK and PK is complicated and not a good pattern.  It obeys  : https://en.wikipedia.org/wiki/KISS_principle

for example, I can not add new enum and get this error:
```
mysql> alter TABLE access_module MODIFY name ENUM('RT_COMPLIANCE_TRAINING', 'DATA_USER_CODE_OF_CONDUCT', 'RAS_LOGIN_GOV', 'RAS_LOGIN_GOV_2');
ERROR 1833 (HY000): Cannot change column 'name': used in a foreign key constraint 'fk_acu_access_module_name' of table 'workbench.access_module_user'

```

I am adding access_module id back as PK for access_module table and FK in this table.